config.yaml

```diff
@@ -41,7 +41,10 @@ rules:
     fastq_screen:
         params: "--force --aligner bowtie2 --subset 100000 --nohits"
         config_file: "../data/fastq_screen.config"
-
+    bbduk_se:
```

Comment:
references should be integrated in the top part of config, we use this section only for parameters

Reply:
apart this, let me say....good job @HyperOdin!